tools/compile_tf_graph.py

```diff
@@ -644,9 +812,10 @@ def cond(i, net_vars, acc_tas, seq_len_info=None)
         seq_len_info = None
       else:
         i, net_vars, acc_tas, seq_len_info = res
-    if seq_len_info:
-      rec_layer.set_state_vars_final_values_recursive(["end_flag", "dyn_seq_len"], seq_len_info)
-    rec_layer.set_state_var_final_value("i", i)
+
+    # make it setable by sprint
+    rec_layer.set_state_var_final_value("global_step", i)
+
```

Comment:
Please undo this change. This should not be needed. Also end_flag, dyn_seq_len should still be there. There can be models depending on that.

Reply:
this is needed now by sprint to control which input to read. end_flag will be useless, because ending logic is handled by sprint. not sure if dyn_seq_len is used at all

Reply:
Both are used by RETURNN, so both are needed.